src/modules/tests/test-data/vehicle-checks/vehicle-checks.cat-be.action.ts

```diff
@@ -0,0 +1,19 @@
+import { Action } from '@ngrx/store';
+import { QuestionResult, QuestionOutcome } from '@dvsa/mes-test-schema/categories/common';
+
+export const SHOW_ME_QUESTION_SELECTED = '[VehicleChecksPage] Show Me Question Selected';
```

Comment:
These actions are in a `cat-be` specific file. In the string would be good to add `cat-be` so that the different categories don't overlap

Reply:
updated action